README.md

```diff
@@ -137,10 +99,53 @@ These development environments should be pre-configured with the same requiremen
 produciton instances, as well as the necessary VS Code extensions required to debug and
 lint code.
 
+## Running locally
```

Comment:
Can we start with `Step 0. Prerequisites` like we have in the API? We'll need to have inframock up and running to do be able to run workers too right?

Reply:
not necessary, you can debug worker-only code without having it connected to the rest of the platform. You can submit custom work to it via another terminal (see the python readme, `pushing custom work to the local worker` section).

Reply:
I removed the prerequisites step, because it was empty